vietanh2810/Sorting-Visualizer
# Add base setup for the project and a .gitignore

## Summary

This merge request resolves issue #2 by creating a base setup for the project

## Changes Made

- created a ReactJS environment 
- add a .gitignore to ignore all the node_modules files

## Verification Steps

To verify that the login form is working correctly, follow these steps:

1. run npm start
2. go to http://localhost:3000/
3. Observe that a React logo is displayed

## Reviewers

@vietanh2810
